framework/encode/vulkan_capture_manager.cpp

```diff
@@ -2822,5 +2823,34 @@ void VulkanCaptureManager::PreProcess_vkBindImageMemory2(VkDevice
     }
 }
 
+void VulkanCaptureManager::PostProcess_vkCreateShaderModule(VkResult                        result,
+                                                            VkDevice                        device,
+                                                            const VkShaderModuleCreateInfo* pCreateInfo,
+                                                            const VkAllocationCallbacks*    pAllocator,
+                                                            VkShaderModule*                 pShaderModule)
+{
+    GFXRECON_UNREFERENCED_PARAMETER(device);
+    GFXRECON_UNREFERENCED_PARAMETER(pAllocator);
+    GFXRECON_UNREFERENCED_PARAMETER(pShaderModule);
+
+    if (result == VK_SUCCESS)
+    {
+        // spirv-parsing for buffer-references
+        gfxrecon::util::SpirVParsingUtil spirv_util;
+
+        if (spirv_util.Parse(pCreateInfo->pCode, pCreateInfo->codeSize))
+        {
+            auto buffer_reference_infos = spirv_util.GetBufferReferenceInfos();
+
+            if (!buffer_reference_infos.empty())
+            {
+                GFXRECON_LOG_WARNING_ONCE(
+                    "Shader is using the 'GL_EXT_buffer_reference2' feature. "
```

Comment:
for VVL, we say things like

`Shader is using the SPV_KHR_physical_storage_buffer (aka GL_EXT_buffer_reference)`

basically use the SPIR-V term incase they aren't using GLSL to create the shader